frontend/src/v5/ui/routes/viewer/tickets/ticketDetails/ticketsDetailsCard.component.tsx

```diff
@@ -73,12 +73,22 @@ export const TicketDetailsCard = () => {
 
 	const onBlurHandler = () => {
 		const values = dirtyValues(formData.getValues(), formData.formState.dirtyFields);
-		let validVals = removeEmptyObjects(nullifyEmptyStrings(filterErrors(values, formData.formState.errors)));
-		validVals = sanitizeViewVals(validVals, ticket, template);
+		const validVals = removeEmptyObjects(nullifyEmptyStrings(filterErrors(values, formData.formState.errors)));
 
-		if (isEmpty(validVals)) return;
+		const editedGroup = findEditedGroup(validVals, ticket, template);
+		if (editedGroup) {
+			TicketsActionsDispatchers.updateTicketGroup(
+				teamspace,
+				project,
+				containerOrFederation,
+				ticket._id,
+				editedGroup,
+				isFederation,
+			);
```

Comment:
Is not necessary to continuing doing stuff. Is one endpoint or the other.
```suggestion
			);
			return;
```

Reply:
The color is a ticket property in the backend, but in the frontend is part of a group. If the user updates the color plus anything else, both the endpoints must be called